### PR TITLE
Editor Sorted by Draft

### PR DESCRIPTION
**Pull request by philsturgeon**
Removed duplicate editors from obsolete list, sorted them by draft support, left
a few implementations on the current list until I hear back from them which
versions they do support, and added Stoplight Studio (which is what I initially
came here to do before I started shaving yaks).

### CODE REVIEW

File: obsolete-implementations.md [hunk 1]
```diff
@@ -195,17 +195,11 @@ Various levels of support for UI generation primarily from the validation vocabu
...
 -   [Visual Studio 2013](http://www.visualstudio.com/) - *Auto-completion and tooltips based on JSON schema draft 3 and draft 4*
--   [JSONBuddy](http://www.json-buddy.com/) - *Grid-style JSON editor and context sensitive entry-helpers based on JSON schema*
--   [ReSharper 2016.1](https://www.jetbrains.com/resharper/) - *code completion, inspections and quick fixes for JSON schema in Visual Studio 2010 - 2015, including support for JSON Path and regular expressions for schema editing*
--   [Visual Studio Code](https://code.visualstudio.com/) - *Schema driven code completion, hovers and validation for editing JSON files (including schemas)*
--   [JSONEditor Online](http://jsoneditoronline.org) - *View, edit, format, and validate JSON online*
 -   [JSON Schema Editor](https://json-schema-editor.tangramjs.com) - *An intuitive editor for JSON schema online*
 -   [JSON Editor](https://json-editor.tangramjs.com) - *An online, schema-aware editor for JSON document*
--   [Eclipse IDE](https://www.eclipse.org/downloads/eclipse-packages) - *Rich JSON edition supporting schema for instantaneous validation and error reporting, completion, documentation.*
```
Comment (philsturgeon) [inline obsolete-implementations.md]: Eclipse is still unknown but was in both lists, so I'm giving it the benefit of the doubt. The rest had draft 4, 6, 7.

**Comment by philsturgeon** [comment]
@jdesrosiers were these the duplicates you noticed in #330? I'd had it sat around pending review for a month 😆 if there's any more let me know.

**Comment by jdesrosiers** [comment]
Nice! You were already on it. That's some of them. I also noticed the "UI Generation" section and the "Online" section. The "Online" section can probably just be removed. I'm pretty sure all "UI Generation" duplicates just belong in implementations, but I didn't check.

**Comment by philsturgeon** [comment]
Did Online efe1515 

There's a TODO for UI Generation, but I've not got time for that one sadly.

**Pull request merged**